### PR TITLE
include templatable in concern

### PR DESCRIPTION
**Pull request by laurajaime**
#### :tophat: What? Why?
In production, `defined?(Decidim::Templates::Admin::Concerns::Templatable)` is nil without include templatable concern

### CODE REVIEW

File: decidim-forms/app/controllers/decidim/forms/admin/concerns/has_questionnaire.rb [hunk 1]
```diff
@@ -16,12 +16,12 @@ module HasQuestionnaire
           included do
             helper Decidim::Forms::Admin::ApplicationHelper
             include Decidim::TranslatableAttributes
+            include Decidim::Templates::Admin::Concerns::Templatable
```
Comment (tramuntanal) [inline decidim-forms/app/controllers/decidim/forms/admin/concerns/has_questionnaire.rb:19]: This will crash if `decidim-templates` is not installed

**Review by tramuntanal — COMMENTED** [review]
@laurajaime I don't think this is the way to fix the problem we're having in an installation with `decidim-templates` installed.

What will happen in installations where `decidim-templates` is not installed?

**Comment by tramuntanal** [comment]
@laurajaime can we close this PR? It's already been fixed.

**Comment by laurajaime** [comment]
> @laurajaime can we close this PR? It's already been fixed.

yes, I close it!

**Pull request closed without merging**